examples/c5g7_transient/rod_translation/assemblies_mg.py

```diff
@@ -0,0 +1,111 @@
+import openmc
```

Comment:
This file is almost identical to `density_change/assemblies_mg.py`. I would just remove it and add a symbolic link to the file in the `density_change` directory.